README.md

```diff
@@ -0,0 +1,94 @@
+# go-billy-gluster [![GoDoc](https://godoc.org/gopkg.in/src-d/go-billy-gluster.v0?status.svg)](https://godoc.org/github.com/src-d/go-billy-gluster) [![Build Status](https://travis-ci.org/src-d/go-billy-gluster.svg)](https://travis-ci.org/src-d/go-billy-gluster) [![codecov.io](https://codecov.io/github/src-d/go-billy-gluster/coverage.svg)](https://codecov.io/github/src-d/go-billy-gluster)
+
+This package provides a basic [go-billy](https://github.com/src-d/go-billy) driver to access [gluster](https://www.gluster.org/) volumes. It uses libgfapi so it's not needed to mount the volume locally.
+
+Currently only basic actions are available, that is:
+
+* `Create`
+* `Open`
+* `OpenFile`
+* `Stat`
+* `Rename`
+* `Remove`
+
+For more information head to the [documentation](https://godoc.org/gopkg.in/src-d/go-billy-gluster.v0)
+
+# Installation
+
+This library uses [gogfapi](https://github.com/gluster/gogfapi) that need libgfapi library to run. This library is contained in the development client packages of gluster.
+
+* Arch / Manjaro:
+
+```
+pacman -S glusterfs
+```
+
+* Ubuntu / Debian:
+
+```
+apt-get install glusterfs-common
+```
+
+* CentOS / RHEL:
+
+```
+yum install glusterfs-api
+```
+
+After installing the dependency you can install the library with `go get`:
+
+```
+go get gopkg.in/src-d/go-billy-gluster.v0
+```
+
+# Example of utilization
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"gopkg.in/src-d/go-billy-gluster.v0"
+)
+
+func main() {
+	fs, err := gluster.New("server", "volume")
+	if err != nil {
+		panic(fmt.Sprintf("cannot connect to gluster volume %s", err))
+	}
+
+	f, err := fs.Create("filename.ext")
+	if err != nil {
+		panic(fmt.Sprintf("cannot create file %s", err))
+	}
+
+	_, err = f.Write([]byte("text"))
+	if err != nil {
+		panic(fmt.Sprintf("cannot write to file %s", err))
+	}
+
+	err = f.Close()
```

Comment:
defer?

Reply:
Even if we would use defer in normal code for an example I find this way easier to understand. There's a clear order closing the file and the filesystem.